activejob/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
-*   Allow `DelayedJob` to report id back to `ActiveJob::Base` as
-    `provider_job_id`.
+*   Allow `DelayedJob`, `Sidekiq` and `que` to report the job id back to
+    `ActiveJob::Base` as `provider_job_id`.
 
     Fixes #18821.
 
-    *Kevin Deisz*
+    *Kevin Deisz* And *Jeroen van Baarsen*
```

Comment:
I'm fine with that!

On Thu, May 7, 2015 at 4:00 PM, Jeroen van Baarsen <notifications@github.com
> wrote:

> In activejob/CHANGELOG.md
> <https://github.com/rails/rails/pull/20056#discussion_r29889974>:
>
> >
> >      Fixes #18821.
> >
> > -    *Kevin Deisz*
> > +    *Kevin Deisz* And *Jeroen van Baarsen*
>
> @kddeisz <https://github.com/kddeisz> you agree with this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/20056/files#r29889974>.
>



-- 
*Kevin D. Deisz*
*TrialNetworks* - part of DrugDev
Software Developer
383 Elliot Street, Suite G
Newton, MA 02464
+1 617.952.4071 x134 (office)
+1 703.615.0396 (mobile)
kdeisz@trialnetworks.com